src/checks/company/name-in-runs.ts

```diff
@@ -0,0 +1,19 @@
+import { CompanyCheck } from '../../types/checks';
+
+const check: CompanyCheck = {
+    id: 'name-in-runs',
+    desc: 'The name of the company cannot be contained in the runs array. It may only contain other services and companies.',
+    url: 'https://github.com/datenanfragen/data/blob/0130049402832ee7ecd0f99986c42aa3334288cd/schema.json#L43',
```

Comment:
The description in the schema doesn't address this issue and linking to a specific commit is unfortunate anyway as we may update that description in the future.

I have expanded the description in the issue a little. Let's link to that instead.

```suggestion
    url: 'https://github.com/datenanfragen/data/issues/2128',
```